### PR TITLE
feat(admin-ui): ui fixes

### PR DESCRIPTION
**Pull request by frdomovic**
# feat(admin-ui): ui fixes

### CODE REVIEW

File: node-ui/src/api/dataSource/AppsDataSource.js [hunk 1]
```diff
@@ -6,6 +6,15 @@ export class AppsDataSource {
   }
 
   async getInstalledAplications() {
-    return Object.keys((await this.client.get("/admin-api/applications")).apps);
+    try {
+      const response = await this.client.get("/admin-api/applications");
+      if (response && response.apps) {
```
Comment (MatejVukosav) [inline node-ui/src/api/dataSource/AppsDataSource.js]: Is this working "response?.apps"

File: node-ui/src/api/dataSource/AppsDataSource.js [hunk 2]
```diff
@@ -6,6 +6,15 @@ export class AppsDataSource {
   }
 
   async getInstalledAplications() {
-    return Object.keys((await this.client.get("/admin-api/applications")).apps);
+    try {
+      const response = await this.client.get("/admin-api/applications");
+      if (response && response?.apps) {
```
Comment (MatejVukosav) [inline node-ui/src/api/dataSource/AppsDataSource.js]: With "?" you are checking if response exists

File: node-ui/src/api/dataSource/AppsDataSource.js [hunk 3]
```diff
@@ -8,7 +8,7 @@ export class AppsDataSource {
   async getInstalledAplications() {
     try {
       const response = await this.client.get("/admin-api/applications");
-      if (response && response?.apps) {
+      if (response && response.apps?.length) {
```
Comment (MatejVukosav) [inline node-ui/src/api/dataSource/AppsDataSource.js]: why not just `response?.apps?.length` ?

Comment (frdomovic) [reply node-ui/src/api/dataSource/AppsDataSource.js]: updated

**Pull request merged**